mne_sandbox/decoding/_time_frequency_csp.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8 -*-
+# Authors: Laura Gwilliams <laura.gwilliams@nyu.edu>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#          Alex Barachant <alexandre.barachant@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from mne.filter import filter_data
+
+
+class TimeFrequencyCSP(object):
```

Comment:
Since this class follows the scikit-learn API, can we make this a subclass of [`sklearn.base.BaseEstimator`](http://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html#sklearn-base-baseestimator) and add the [`sklearn.base.TransformerMixin`](http://scikit-learn.org/stable/modules/generated/sklearn.base.TransformerMixin.html#sklearn-base-transformermixin)?